### PR TITLE
Add: Config-Placeholders-Freshservice sample app

### PR DESCRIPTION
**Pull request by teja-kummarikuntla**
A sample app for Freshservice presenting all the UI-enabled app locations with a placeholder.
The app locations are taken from the Freshservice developer documentation.

**App Locations:**
1. Global Navigation Pane
    - `full_page_app `
2. Ticket Details Page
    - `ticket_sidebar `
    - `ticket_requester_info `
    - `ticket_top_navigation`
3. Change Details Page
    - `change_sidebar`
4. Contact Details Page
    - `contact_sidebar `
5. New Ticket Page
    - `new_ticket_sidebar `
6. New Change Page
    - `new_change_sidebar `
7. Asset Details Page
    - `asset_sidebar `

### CODE REVIEW

File: Freshworks-Samples/App-Development-Features/Configuration-Features/config-placeholders-freshservice/app/styles/images/aeroplane.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 100 100"><text x="50%" y="50%" dominant-baseline="central" text-anchor="middle" font-size="64">✈️</text></svg>
```
Comment (Saif-Shines) [inline Freshworks-Samples/App-Development-Features/Configuration-Features/config-placeholders-freshservice/app/styles/images/aeroplane.svg:1]: I pick icons from this website. 

If you find aeroplane boring, you can pick an alternative one from here - https://formito.com/tools/favicon?ref=jonas.io

File: Freshworks-Samples/App-Development-Features/Configuration-Features/config-placeholders-freshservice/app/styles/images/icon.svg [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+			<g id="Group-7">
+				<path id="FS-Due-Icon" class="st0" d="M31.6,60.3C16.3,60.3,4,48,4,32.7S16.3,5,31.6,5h20.8c3.9,0,7,3.1,7,7v20.7
+					C59.3,48,47,60.3,31.6,60.3z M31.2,46.1c0.3,0.3,0.8,0.2,1.1-0.2l8.8-12.2l1.9-2.8c0.5-0.6,0-1.6-0.8-1.6h-3.9h-9.2l3.2-9.8
+					c0.2-0.3,0-0.7-0.3-0.8c-0.3-0.3-0.8-0.2-1.1,0.2l-8.8,12.2l-1.9,2.6c-0.5,0.7,0,1.6,0.8,1.6h3.9h9.2l-3.2,9.8
+					C30.9,45.5,30.9,46,31.2,46.1z"/>
+			</g>
+		</g>
+	</g>
+</g>
+</svg>
```
Comment (Saif-Shines) [inline Freshworks-Samples/App-Development-Features/Configuration-Features/config-placeholders-freshservice/app/styles/images/icon.svg:22]: I have noticed that this icon is no longer used for the app. You can safely delete this.

**Review by Saif-Shines — CHANGES_REQUESTED** [review]
There are some non blocking suggestions if you plan to incorporate. Do you plan to incorporate them?

File: Freshworks-Samples/App-Development-Features/Configuration-Features/config-placeholders-freshservice/app/styles/images/aeroplane.svg (hunk 1 above)
Comment (teja-kummarikuntla) [reply Freshworks-Samples/App-Development-Features/Configuration-Features/config-placeholders-freshservice/app/styles/images/aeroplane.svg:1]: I tried with a few puppies. Nonetheless, Planes and Copters looks the best either.

**Comment by teja-kummarikuntla** [comment]
> There are some non blocking suggestions if you plan to incorporate. Do you plan to incorporate them? Let me know if you plan to incorporate them in a next cycle or so. I will have this PR approved.

Yes, let me know, we can work that!

**Review by Saif-Shines — DISMISSED** [review]

File: Freshworks-Samples/App-Development-Features/Configuration-Features/config-placeholders-freshservice/app/views/asset_details_page/asset_sidebar.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    function whenInteractive() {
+      if (document.readyState === 'interactive') {
+        return app.initialized().then(getClientAPI).catch(console.error);
+      }
+    }
+
+    function getClientAPI(_client) {
+      let client = _client;
+      client.events.on('app.activated', function log() {
+        return console.log(`asset sidebar location: app is activated 🥁`);
```
Comment (ravirajsubramanian) [inline Freshworks-Samples/App-Development-Features/Configuration-Features/config-placeholders-freshservice/app/views/asset_details_page/asset_sidebar.html:27]: Please convert any console.log statement to console.info if they are important to understand the flow and console.error if they are for error handling.

File: Freshworks-Samples/App-Development-Features/Configuration-Features/config-placeholders-freshservice/app/views/asset_details_page/asset_sidebar.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      });
+    }
+  </script>
+</html>
```
Comment (ravirajsubramanian) [inline Freshworks-Samples/App-Development-Features/Configuration-Features/config-placeholders-freshservice/app/views/asset_details_page/asset_sidebar.html:31]: Please add an empty line at the end of the file.

**Review by ravirajsubramanian — DISMISSED** [review]

**Review by Saif-Shines — APPROVED** [review]
🏄‍♀️

**Pull request merged**